requirements.txt

```diff
@@ -4,3 +4,4 @@ numpy
 scipy
 pytest
 flake8
+networkx
```

Comment:
I think this is a considerable requirement!

Reply:
sure, agreed, no need to add it, I can just delete the function that does `Graph` out of it... if anyone needs it, they can do it afterwards by themselves!